apps/site-landing/src/containers/App/Routes.tsx

```diff
@@ -16,7 +15,6 @@ import { Privacy } from 'containers/Privacy';
 export const Routes = () =>
   <Switch>
     <Route path="/" exact component={HomePage} />
-    <Route path="/newsletter/confirm" exact component={Confirm} />
```

Comment:
This is a questionable decision - It would be nice to host the confirmation page in our website (rather than on mailjet).

Unfortunately, it's probably just so far down the list of priorities that by the time I get to it this code branch would be rotted and more hassle than it's worth